proto/substrait/algebra.proto

```diff
@@ -228,6 +228,19 @@ message AggregateRel {
   }
 }
 
+message WindowRel {
```

Comment:
Probably useful to add comments for the various parts.  I'm especially curious how this differs from the FilterRel so that might be useful information to provide here.